### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by CaselIT**
These are similar to the sqlalchemy ones

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is. -->
+
+**Expected behavior**
+<!-- A clear and concise description of what you expected to happen. -->
+
+**To Reproduce**
+Please provide a [Minimal, Complete, and Verifiable](http://stackoverflow.com/help/mcve) example. 
+See also [Reporting Bugs](https://www.sqlalchemy.org/participate.html#bugs) on the website.
```
Comment (zzzeek) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: alembic is not as easy here but yes, we should encourage them to try :)

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: requires triage
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (zzzeek) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]: this is good.

File: .github/ISSUE_TEMPLATE/question.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+name: Question
+about: Question regarding Alembic features
+title: ''
+labels: requires triage
+assignees: ''
+
+---
+
+**Describe you question**
```
Comment (zzzeek) [inline .github/ISSUE_TEMPLATE/question.md:10]: typo, "your"

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (CaselIT) [reply .github/ISSUE_TEMPLATE/bug_report.md:18]: I'll try to clarify a bit, to suggest to at provide the migration that causes problems and the sqlalchemy table/model

File: .github/ISSUE_TEMPLATE/question.md (hunk 3 above)
Comment (CaselIT) [reply .github/ISSUE_TEMPLATE/question.md:10]: Good catch, I think it's also in sqlalchemy. Will correct

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (CaselIT) [reply .github/ISSUE_TEMPLATE/feature_request.md:11]: this was suggested by the github template :)

**Pull request merged**